pkg/appmanager/resourceConfig_test.go

```diff
@@ -659,6 +662,41 @@ var _ = Describe("Resource Config Tests", func() {
 				Expect(cfg.Virtual.VirtualAddress.BindAddr).To(Equal("5.6.7.8"))
 			})
 
+			It("handles invalid source address translation annotations for ingress", func() {
```

Comment:
@russokj here is the garbage input unit test I added as per Booch's comment yesterday feel free to review at your leisure.